bricks/template/__brick__/{{project_name.snakeCase()}}/ios/Podfile.lock

```diff
@@ -31,4 +31,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 22cb4141f6291ab800e0ca928cc1f87e9a69bf64
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.11.3
```

Comment:
@chornerman it is not a big due, could we update the latest version? If not, please do not make the change 🙏

Reply:
@doannimble I do not update it manually. It's auto-generated when we run the project on iOS. So I think we should leave it as is 🙏🏼